LEAF_Nexus/scripts/refreshOrgchartEmployees.php

```diff
@@ -243,20 +254,33 @@ function updateEmployeeDataBatch(array $localEmployeeUsernames = [])
     if (empty($orgEmployeeDataRes)) {
         return FALSE;
     }
+
+    // regather employee data since we may have inserted items
+    list($localEmpUIDs,$localEmpArray) = getEmployeeUIDs($localEmployeeUsernames);
+
     foreach ($orgEmployeeDataRes as $orgEmployeeData) {
 
-        $localEmployeeDataArray[] = [
-            // this will need to be checked into further I cannot silence this warning without possibly breaking things.
-            'empUID' => $localEmpArray[$orgEmployeeData['userName']],
-            'indicatorID' => $orgEmployeeData['indicatorID'],
-            'data' => $orgEmployeeData['data'],
-            'author' => $orgEmployeeData['author'],
-            'timestamp' => $orgEmployeeData['timestamp'],
-        ];
-    }
+        // if this user is not found, we will skip adding data for them.
+        if(empty($localEmpArray[$orgEmployeeData['userName']]) == true ){
+            continue;
+        }
+        else{
+            $localEmployeeDataArray[] = [
+                'empUID' => $localEmpArray[$orgEmployeeData['userName']],
+                'indicatorID' => $orgEmployeeData['indicatorID'],
+                'data' => $orgEmployeeData['data'],
+                'author' => $orgEmployeeData['author'],
+                'timestamp' => $orgEmployeeData['timestamp'],
+            ];
+        }
 
-    $oc_db->insert_batch('employee_data',$localEmployeeDataArray,['indicatorID','data','author','timestamp']);
+        
+    }
 
+    // make sure data array has data before attempting to insert data
+    if(empty($localEmployeeDataArray) == false){
```

Comment:
Same here, do we need the == false?

Reply:
removed false, had it since it made it very clear we are looking at true/false.